### ISSUE TITLE
import https://github.com/github/linguist/blob/master/lib/linguist/languages.yml to Languages difficulty/medium

### ISSUE THREAD

**Issue by co-robo** [coala/coala#4556]
Opened by @jayvdb at [gitter/coala/coala](https://gitter.im/coala/coala?at=5976d1c989aea4761dcfa60e)

**Comment by adtac** [comment]
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

**Comment by RaiVaibhav** [comment]
@jayvdb @adtac @Makman2  But i think we can extend only `aliases` and `extensions` from `linguist/languages.yml` to https://github.com/coala/coala/tree/master/coalib/bearlib/languages/definitions i.e to each `LANGUAGE`  ie. `C.py, CPP.py .......`  or correct me if i am missing something?

**Comment by adtac** [comment]
@RaiVaibhav can you please not mention three unrelated users on issue comments? This is not the first time (see [this](https://github.com/coala/coala-quickstart/issues/62#issuecomment-338500520), [this](https://github.com/coala/coala-quickstart/issues/62#issuecomment-338682547), this issue) :/

I get subscribed to discussions I don't want to be a part of (mostly because I know next to nothing on those topics). Just comment on the issue and someone will eventually get back to you. If not, post on gitter. Again, without mentioning anyone in particular. The obvious exception is if a person really needs to be mentioned.

**Comment by RaiVaibhav** [comment]
@adtac :+1 Next time i will remember it,you are the owner so i thought   may be you might help me.

**Comment by adtac** [comment]
Nope, jayvdb opened the issue. See the first comment. I merely changed the title (because links in titles aren't good).

**Comment by shreyans800755** [comment]
@jayvdb 
Can you please give more details ? I would like to work on this issue.
Is this about using the mentioned yaml file to derive the language in which input file was written ?
Is there any discussion on gitter after which this issue was filed ?
